response-to-reviewers.tex

```diff
@@ -536,21 +542,31 @@ \subsection*{Random thoughts I had while reading the paper, to discuss (or
 with start\_time.
 \end{point}
 \begin{reply}
-TODO
-
-\ggcomment{
-\begin{itemize}
-\item
-We provide parsers, so the simulation software doesn't have to
-write this. Thus parsers are shared between simulators.
-\item
-The parser does validation too.
-\item
-A simulator can choose to only accept MDM models.
-\end{itemize}
-}
-
+It's not necessarily true that a program that accepts the MDM will need to
+write its own Demes (MDM) parser. For example, msprime uses the \texttt{demes}
+Python module to accept demographic models in Demes format. In effect,
+what msprime does is take the object model provided by the parser
+which is identical to an MDM document. Msprime then processes this document,
+converting it into its own internal Demography format. Once the conversion
+into msprime's internal format is complete, \emph{this} object
+model is validated using precisely the same code as if the model
+had been produced by msprime's own demography APIs.
+The only validation that msprime does on the Demes MDM document
+is to check for parameters that it doesn't support (like selfing and cloning
+rates).
+
+Thus, from msprime's perpective, there is no difference between what we
+have currently (the HDM converted to MDM via the parser) and what
+you propose (the MDM validated by the parser).
+
+If the HDM did not exist, we would still require parsers because validating
+the full MDM is a lengthy and tedious task.
+A key goal that we set was for the ecosystem to be ``batteries-included''
+from the outset, which is why we have implemented several high-quality
+parsers (Python, and C with Rust, Julia in development) which should
```

Comment:
The rust parser is out and ready for use.

Reply:
I'll take care of this on my round of edits!